src/pages/groupComparison/MultipleCategoryBarPlot.tsx

```diff
@@ -424,12 +435,56 @@ export default class MultipleCategoryBarPlot extends React.Component<
     }
 
     @computed get labels() {
+        if (this.props.sortOption == 'sortByCount') {
```

Comment:
@SURAJ-SHARMA27  this is going to actually mutate the sorting of counts in ```this.data```.  i.e. sort it in place.  instead, we should use functional/immutable approach to return a sorted list of labels without mutating underlying data.